### PR TITLE
Add a 'current-work' location to allow people to highlight current areas of focus or interest

### PR DESCRIPTION
**Pull request by Relequestual**
Resolves https://github.com/json-schema-org/community/issues/249

**GitHub Issue:** https://github.com/json-schema-org/community/issues/249

**Summary**: Add two files, one which explains the purpose of logging current areas of interest, and another to contain them.

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

No
It's a new thing we are testing which is open to change based on feedback.

### CODE REVIEW

File: current-work/README.md
Comment (karenetheridge) [inline current-work/README.md:20]: the grammar here isn't quite right

**Comment by Julian** [comment]
(At the high level certainly seems like a reasonable thing to do given we're working full-time)

Re: "too-manual" which apparently was brought up on the call that discussed this -- how often was the thought to update this / how granular of info were we going to give? If it's "once every few weeks with a mid-level area of focus" that seems ok to me to self-report (though of course tools may be nice for people too). If it's more granular ("what did you work on yesterday") doing that manually does sounds tedious and may mean we each *need* to use one (or agree on a collective one).

For an opinion -- if we're testing this out and trying to just make sure we're all open about areas we're working on, seems we certainly could start with more of the former than the latter.

**Comment by handrews** [comment]
@Julian it's intended to be a top-3 thing so that folks who aren't necessarily watching every notification can check and see what's worth looking at without slogging through 200 emails.

So right now my top 3 would be the [JRI PR](https://github.com/json-schema-org/referencing/pull/2) in the referencing repo, the "[Making annotation collection practical](https://github.com/orgs/json-schema-org/discussions/236)" discussion, and the "[Behavior negotiation: Balancing author intent and runtime flexibility](https://github.com/orgs/json-schema-org/discussions/243)" discussion,  I might make the JRI entry just point to the repository or to issues/PRs there with the "jri" tag so I don't have to change it when the JRI PR is merged.

So I'd say probably a weekly update at most, but as you can see all of mine are more than a week old, and two of them are much older.

**Comment by handrews** [comment]
In a rarity for me, I think for this it is better to just direct-commit.  I suppose there's a chance of accidentally deleting someone's entry, but that should be noticed fairly quickly and isn't the end of the world by any stretch.  We don't really need the noise of nudging each other to review/merge PRs on things that we are defining for ourselves.  Let's just pick an order (alphabetical by.. first name? last name?  github username?) and update the file directly unless/until that causes problems.

**Comment by Relequestual** [comment]
My expectation is this would only be updated maybe once or twice a month. Thinking about the bigger items as opposed to small detail. For example, "Focusing on fixing issues in my implementation" vs "Fixing these specific GitHub issues 1,2,3".

However, the specific Issues may matter if they are bigger or require many sub-issues.

**Comment by Relequestual** [comment]
> I think for this it is better to just direct-commit.

OK, I'll modify the language. I think I agree.
Anyone who's "part of the core team" can commit directly. Anyone who isn't, should they want to add to the list, may make a PR.

**Review by Julian — APPROVED** [review]

File: current-work/README.md
Comment (handrews) [inline current-work/README.md:6]: Can we change "log their current areas of focus or interest" to "indicate which areas of their work most require attention from the core team members"?

File: current-work/README.md
Comment (handrews) [inline current-work/README.md:10]: I'd change 'focus on the "important things"' to 'respond to the most immediately important things' or something of that sort.  This is about responding to each other, rather than priorities in general.

File: current-work/README.md
Comment (handrews) [inline current-work/README.md:21]: Remove "either".  We should probably emphasize that pushing commits to this repo for any other file remains (forbidden? discouraged?  subject to process X? idk what the right statement is).

File: current-work/README.md
Comment (handrews) [inline current-work/README.md:33]: Is there a situation where this is anything other than "look at it and respond"?  This question feels like unnecessary boilerplate.

**Review by handrews — CHANGES_REQUESTED** [review]
I have a few minor wording suggestions based on our call earlier this week.

File: current-work/README.md
Comment (Relequestual) [reply current-work/README.md:33]: Well, it could be you want someone to review the PR, or maybe you just want thoughts so far. Maybe you're looking to tackle one last bit relating to a specific comment. If it's a Discussion, it might be a specific thread you want people to reply to. IDK. Assumptions bad.
If we find it not useful, we can remove it later.

Comment (handrews) [reply current-work/README.md:33]: If I want something specific looked at I'll say so in the entry.  This line just feels like an annoyance and I don't plan to include it in my entries.  This is supposed to be the easiest possible process for us to draw attention however we need it.  I really don't like boilerplate - I'm not going to block the PR over it, but I don't like it.

**Review by handrews — APPROVED** [review]
I'm fine with this as-is, thanks!

**Pull request merged**
